dts/bindings/ieee802154/atmel,rf2xx.yaml

```diff
@@ -38,3 +38,53 @@ properties:
       description: |
         Specifies the MAC address that was assigned to the network
         device
+
+    channel-page:
+      type: int
+      enum:
+        - 0
+        - 2
+        - 5
+      default: 0
+      description: |
+        Selects Channel Page accordingly with IEEE 802.15.4 standard. The Page 0
+        is used in both Sub-Giga and 2.4GHz. It allows select channels 0-10 in
+        Sub-Giga band (0: BPSK-20, 1-10: BPSK-40) and 11-26 in 2.4GHz band
+        (11-26: O-QPSK-250). Channel 2 is for Sub-Giga and selects
+        (0: OQPSK-SIN-RC-100, 1-10: OQPSK-SIN-250). Channel 5 is for Sub-Giga
+        (JAPAN) and selects (0-3: OQPSK-RC-250) .
+          0: Page 0 - BPSK-20 [0], BPSK-40 [1-10], O-QPSK-250 [11-26].
+          2: Page 2 - OQPSK-SIN-RC-100 [0], OQPSK-SIN-250 [1-10].
+          5: Page 5 - OQPSK-RC-250 [0-3].
+
+    tx-pwr-map:
+      type: uint8-array
+      default: [ 0x00 ]
+      description: |
+        This is the Transmission Power Mapping array used to comply with local
+        regulations. By default this value set an output power above 0dBm for all
+        transceivers. This property must be used with tx-pwr-min and tx-pwr-max
+        for normal operations. The max size of array is defined by
+        CONFIG_IEEE802154_RF2XX_TX_PWR_MAP_ENTRIES Kconfig property. The max
+        value allowed for 2.4GHz is 0x0f and for Sub-Giga 0xff. See PHY_TX_PWR
+        at datasheet for more details.
+
+    tx-pwr-min:
+      type: uint8-array
+      default: [ 0x00, 0x00 ]
+      description: |
+        This value represent minimum normalized value in dBm for the transceiver.
+        This property must be used with tx-pwr-mapping. Note that this allows
+        use of external transceiver. The value is represented by two values where
+        first represent signal indication and second is the value itself. The
+        default value is set equal to tx-pwr-max to create a table with one entry.
```

Comment:
which table has one entry in it?  The last sentence is confusing.

Reply:
Add a more detailed description.